packages/react/src/FilteredActionList/FilteredActionList.tsx

```diff
@@ -9,7 +9,7 @@ import type {TextInputProps} from '../TextInput'
 import TextInput from '../TextInput'
 import {get} from '../constants'
 import {ActionList} from '../deprecated/ActionList'
-import type {GroupedListProps, ListPropsBase} from '../deprecated/ActionList/List'
+import type {GroupedListProps, ListPropsBase} from '../SelectPanel/types'
```

Comment:
Should we bring FilteredActionList into src/SelectPanel, it's not used anywhere else. (And we probably don't want it to be used anywhere else?)

Reply:
That makes sense to me! Perhaps in a follow-up PR?

Reply:
Hmm, I think we exported it recently? https://github.com/primer/react/pull/4816

Reply:
I am merging this to un-complicate [my follow up select panel PR](https://github.com/primer/react/pull/479) but please let me know what you think here so that I can address things in a follow up PR if needed.